UI/aoc-app/src/App.js

```diff
@@ -1,13 +1,34 @@
 import React from 'react';
 import './App.css';
+import {
+  BrowserRouter as Router,
+  Redirect,
+  Route,
+  Switch,
+} from "react-router-dom";
+import CreateProfile from "./Components/CreateProfile";
+import HomePage from "./Components/HomePage";
+import SignUpForm from "./Components/SignUpForm";
 
 
 // rendering app
 function App() {
   return (
-    <div className="App">
-      Hello World!   
-    </div>
+    <Router>
+      <div className="App">
+        <Switch>
+          <Route exact path="/">
+              <HomePage/>
+            </Route>
+            <Route exact path="/createProfile">
+              <CreateProfile />
+            </Route>
+            <Route exact path="/sign-up">
```

Comment:
Let's make the routes standardized to either kebab-case or camel-case. Either is fine, but they should follow the same convention

Reply:
Yeah, that's a good point. According to this stack link, hyphenated is the way the go: 
https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris

Reply:
I think that we should add a story to our backlog  for a little refactor for all of this route stuff, i have a few ideas